src/Conf.ml

```diff
@@ -1509,13 +1519,21 @@ let parse_line config ~from s =
               , Format.sprintf "expecting %s but got %s" Version.version
                   value ))
     | name, `File x ->
-        C.update ~config
-          ~from:(`Parsed (`File x))
-          ~name ~value ~inline:false
+        if !disable_conf_files then
+          Error (`Malformed "Disabled file configuration of ocamlformat")
+        else
+          C.update ~config
+            ~from:(`Parsed (`File x))
+            ~name ~value ~inline:false
     | name, `Attribute ->
-        C.update ~config
-          ~from:(`Parsed `Attribute)
-          ~name ~value ~inline:true
+        if !disable_conf_attrs && not (String.equal name "disable") then
+          Error
```

Comment:
This error is not shown to the user and attributes are ignored.

Reply:
fixed